sevntu-checks/src/test/resources/com/github/sevntu/checkstyle/checks/coding/InputJsr305AnnotationsCheckWithNestedAnnotation.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I don't think we need this license in the input file.

Reply:
Ah, copy-and-paste devil. Done.